CONTRIBUTING.md

```diff
@@ -134,6 +134,7 @@ The conventions for module exports are:
 ### Version Control Conventions
 
 * We use [rebase merging](https://git-scm.com/book/en/v2/Git-Branching-Rebasing) (as opposed to [basic merging](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging#Basic-Merging)) to merge branches
+* To rebase your fork's PR branch onto master, the recommended approach is `git rebase origin/master`, `git push --force origin your-branch`
```

Comment:
I think this is helpful, but do you think we should also drop a note about branch names? eg. it's not clear what origin is here when you have a fork depending how you've setup your local repo, since you'll likely have one remote for your fork and a remote for upstream (mapbox/mapbox-gl-js)

Reply:
Yeah good point it should really be `upstream/master` for most people... Maybe it's worth saying full details on the "preferred setup", e.g.
1. Fork this project
2. Clone your new fork, `git clone git@github.com:GithubUser/mapbox-gl-js.git`
3. `cd mapbox-gl-js`
4. Add the Mapbox repository as upstream: `git add remote upstream git@github.com:mapbox/mapbox-gl-js.git`
5. Create a new branch `git checkout -b your-branch` for your contribution
6. Write code
7. When you need to rebase, `git fetch upstream`, `git rebase upstream/master` and force push to Github `git push --force origin your-branch`

Reply:
It would be sufficient in this instruction to add a note for downstream forks:
```suggestion
* To rebase your PR branch onto master, the recommended approach is `git rebase origin/master`, `git push --force origin <your-branch>`. When working with a fork, `origin/master` should point to the upstream `master` branch.
```